src/main/kotlin/com/hibob/academy/dao/OwnerDao.kt

```diff
@@ -0,0 +1,50 @@
+package com.hibob.academy.dao
+
+import org.jooq.DSLContext
+import org.jooq.Record
+import org.jooq.RecordMapper
+import org.springframework.stereotype.Component
+import java.math.BigInteger
+import java.time.LocalDateTime
+import java.util.Date
+
+data class Owner(val name: String, val companyId: Long, val employId: String)
+
+
+@Component
+class OwnerDao(private val sql: DSLContext) {
+    private val table = OwnerTable.instance
+
+
+    private val ownerMapper = RecordMapper<Record, Owner> { record ->
+        Owner(record[table.name], record[table.companyId], record[table.employId])
+    }
+
+    fun getOwners(): List<Owner> =
+        sql.select(table.name, table.companyId, table.employId)
+            .from(table)
+            .fetch(ownerMapper)
```

Comment:
what about companyId?
We fetch by companyId from the db.
Get the companyId as param and use it in the where.